.kitchen.dokken.yml

```diff
@@ -26,20 +26,25 @@ platforms:
     pid_one_command: /bin/systemd
 
 suites:
```

Comment:
All of the dokken suites can be removed as we use KITCHEN_LOCAL to override certain sections. We'll just manage suites in the main kitchen.yml

Reply:
Cool, that is a lot of duplicate work.

Reply:
Done.